components/worktemplates/components/menu/use_case.tsx

```diff
@@ -100,63 +102,86 @@ const StyledUseCaseMenuItem = styled(UseCase)`
     display: flex;
     flex-direction: column;
     width: 220px;
-    height: 185px;
     border: 1px solid rgba(var(--center-channel-text-rgb), 0.16);
     border-radius: 8px;
     cursor: pointer;
     margin-bottom: 16px;
 
     ${QuickUse} {
-        display: none;
+        visibility:hidden;
+        opacity:0;
     }
 
     .illustration {
-        height: 129px;
+        height: 130px;
         background: rgba(73, 146, 243, 0.2);
         border-radius: 8px 8px 0px 0px;
         display: flex;
         align-items: flex-end;
         justify-content: center;
         position: relative;
+        flex-grow: 1;
+        overflow-x: hidden;
+        overflow-y: hidden;
+        transition: height 0.4s ease-in-out;
 
         img {
             width: 204px;
             height: 123px;
+            z-index: 1;
+            transition: margin 0.4s ease-in-out;
         }
     }
 
     .name {
```

Comment:
Please make sure to match styles for this section from figma design in all the states: https://www.figma.com/file/xS2v61Rlq2DiLG8QF3fIPU/MMO-MVP?node-id=592%3A194495